extensions/cornerstone/src/utils/WarningsViewportOverlay.js

```diff
@@ -0,0 +1,236 @@
+import { PureComponent } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import cornerstone from 'cornerstone-core';
+import './WarningsViewportOverlay.css';
+import moment from 'moment';
+import classNames from 'classnames';
+import { Icon } from '@ohif/ui/src/elements/Icon';
+import { Tooltip } from '@ohif/ui/src/components/tooltip';
+import { OverlayTrigger } from '@ohif/ui/src/components/overlayTrigger';
```

Comment:
This component doenst fit well in utils, move it to a components folder under cornerstone extension.

Reply:
done